src/types/onyx/Report.ts

```diff
@@ -12,6 +12,15 @@ type Report = {
     /** List of icons for report participants */
     icons?: OnyxCommon.Icon[];
 
+    /** Are we loading more report actions? */
+    isLoadingOlderReportActions?: boolean;
+
+    /** Are we loading newer report actions? */
+    isLoadingNewerReportActions?: boolean;
+
+    /** Flag to check if the report actions data are loading */
+    isLoadingInitialReportActions?: boolean;
+
```

Comment:
Revert? Those props are for ReportMetadata only

Reply:
yep, didn't catch that, thanks!